templates/resolv.conf.tmpl

```diff
@@ -32,7 +32,7 @@ options
  {{flag-}}
 {% endfor %}
 
-{%- for key, value in options.items() %}
+{%- for key, value in options.items()|sort %}
```

Comment:
Were you seeing failures outside of CI runs?

Reply:
Locally I'm running a version of Python where insertion ordering of dicts is retained, so I couldn't reproduce the failure locally. xenial's Python doesn't retain insertion order, so the order was different there (due to key randomisation).

Reply:
Sure, I guess I was wondering if our existing CI checks found this or you found it manually somewhere else.

Reply:
Oh right, yep, our xenial CI run was failing before this change.

Reply:
Specifically: https://travis-ci.org/github/canonical/cloud-init/jobs/661725510?utm_medium=notification&utm_source=github_status

Reply:
Excellent!  That's what was hoping for.  Thanks for the confirmation.